test-data/unit/fixtures/implicit_class_method.pyi

```diff
@@ -0,0 +1,13 @@
+# builtins stub with object.__init_subclass__ and __class_getitem__
+
+class object:
+    def __init_subclass__(cls) -> None: pass
+
+    def __class_getitem__(cls, item) -> None: pass
```

Comment:
I think that `object` doesn't define `__class_getitem__`, so this method should probably be in some other class. Perhaps add a base class in your test case? If you do this, it looks like you don't need to touch the fixtures at all.